SpokestackFrameworkExample/CoreMLViewController.swift

```diff
@@ -88,6 +88,9 @@ class CoreMLViewController: UIViewController {
         }
         c.detectModelPath = detectPath
         c.tracing = Trace.Level.PERF
+        c.melFrameLength = 400
+        c.wakewords =  "up,dog"
+        c.wakePhrases = "up dog"
```

Comment:
This summer when I was testing the wake work stuff I said `up dog` so many times my youngest daughter would start to repeat it.